BigBaseV2/src/backend/looped/vehicle/keep_vehicle_repaired.cpp

```diff
@@ -0,0 +1,13 @@
+#include "backend/looped/looped.hpp"
+#include "util/vehicle.hpp"
+
+namespace big
+{
+	void looped::vehicle_keep_vehicle_repaired()
+	{
+		bool toggle = VEHICLE::GET_DOES_VEHICLE_HAVE_DAMAGE_DECALS(self::veh);
+		if (g->vehicle.keep_vehicle_repaired && toggle) {
```

Comment:
Can this native call be inlined in the if statement? That way we wouldn't be making unnecessary native calls if the keep_vehicle_repaired isn't set to true.